bss.cc

```diff
@@ -555,6 +555,34 @@ trefethen4(const std::vector<double>& xs)
     return ret;
 }
 
+//Valley Functions
+double
+three_hump_camel_back(const std::vector<double>& xs)
+{
+    double x1 = xs.at(0);
+    double x2 = xs.at(1);
+
+    double ret = (2 * SQ(x1)) 
+        - (1.05  * std::pow(x1, 4))
+        + (std::pow(x1, 6) / 6)
+        + (x1 * x2)
+        + SQ(x2);
+    return ret;
+}
+
+double 
+dixonPrice(const std::vector<double>& xs)
```

Comment:
this function name still mixes cases, `dixon_price` would be consistent with the rest